### DISCUSSION TITLE
Question re Volume Mapping when using docker-compose.yml

### DISCUSSION THREAD

**Discussion by jenskock** [photoprism/photoprism#2021] [Q&A] [answered]
Folks, I am trying to setup a PhotoPrism environment with docker-compose. Basically pp ist running like expected but the folders that are mapped under volumes are always empty. I've just uploaded some photos via the web interface and thought that this photos are stored on the host in the mapped "originals" folder. Also the storage folder is empty.

After that I've run a ```docker system prune``` and ```docker volume prune``` After firing up another container the images were still present. Now I am completely lost.

```yaml
volumes
  - "./pictures:/photoprism/originals"
  - "./storage:/photoprism/storage"
```
<img width="202" alt="Screenshot 2022-02-07 at 21 25 32" src="https://user-images.githubusercontent.com/10282748/152866136-95082bc1-7424-474f-a668-3eb8e66caf47.png">

**Comment by lastzero** [comment]
/etc is definitely the wrong base folder to store your files, it's for config files only. You can either use the home directory of your current user or, for example, run photoprism in /opt/photoprism. Note the originals folder must be mounted, looks like you might have mounted pictures instead.

**Reply by jenskock** [reply]
I didn't know that the volumes can't be folders, so I think that's the problem.

**Comment by girlpunk** [comment]
> folders that are mapped under volumes are always empty

Are you sure you're using volumes, because that syntax looks like folder mounts instead

**Reply by jenskock** [reply]
Nope I've used folders.

**Comment by jenskock** [comment, marked as answer]
It is possible to mount folders. The cause was the docker installation that comes with ubuntu server via snap. See [this](https://stackoverflow.com/questions/45764477/docker-compose-error-while-creating-mount-source-path) stack overflow thread for details.

**Reply by girlpunk** [reply]
Folder mounts are indeed a thing, but they're different to volumes. [Take a look at this for more information](https://stackoverflow.com/a/49173474)
